Client/Impl/Worker/JobWorkerBuilder.cs

```diff
@@ -102,6 +103,18 @@ public IJobWorkerBuilderStep3 FetchVariables(params string[] fetchVariables)
         public IJobWorkerBuilderStep3 PollInterval(TimeSpan pollInterval)
         {
             this.pollInterval = pollInterval;
+            // For backward compatibility
+            if (this.reconnectInterval == default(TimeSpan))
+            {
+                this.reconnectInterval = pollInterval;
+            }
+
```

Comment:
:+1: thought the same :)

Reply:
Yep. But some of the workers must work very quickly ( 
I think it would be enough to change from pooling interval to 
```
catch (RpcException rpcException)
{
    LogRpcException(rpcException);
    await Task.Delay(TimeSpan.FromMilliseconds(activateJobsRequest.RequestTimeout), cancellationToken);
}
```
Maybe this is better than adding the code that I wrote in this pull request.

What do you think?

Reply:
By the way, i think that isRunning have to be false in this case.